app/code/LongHoang/VnpayWallet/Gateway/Helper/ResponseMessage.php

```diff
@@ -18,56 +53,56 @@ class ResponseMessage
     public function getErrorMess(string $responseCode)
     {
         switch ($responseCode) {
-            case "00":
-                $result = "Giao dịch thành công";
+            case self::RES_CODE_00:
+                $result = __("Giao dịch thành công");
```

Comment:
Translate thì văn bản gốc phải bằng tiếng anh chứ em? 
```suggestion
                $result = __("Transaction complete");
```
Ví dụ